gobcore/message_broker/async_message_broker.py

```diff
@@ -205,7 +205,7 @@ def publish(self, queue, key, msg):
             raise Exception("Connection with message broker not available")
 
         # Convert the message to json
-        json_msg = json.dumps(msg, cls=GobTypeJSONEncoder)
+        json_msg = json.dumps(msg, cls=GobTypeJSONEncoder, allow_nan=False)
```

Comment:
If I test this locally in Import-Client I'm getting the following error:
ValueError: Out of range float values are not JSON compliant